### PR TITLE
Add java-lite and generators

### PR DESCRIPTION
**Pull request by tumarsal**
Add java-lite 
Add generators for java-lite and java code

### CODE REVIEW

File: grpc/Makefile [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Copyright (2019) Cobalt Speech and Language Inc.
 
-.PHONY: all go py cs swift cs_push android go-test py-test docs
+.PHONY: all go py cs swift cs_push android java-lite java go-test py-test docs
 
 all: docs go py swift
 
 ifeq ($(PROTOINC),)
- $(error "env variable PROTOINC must be defined to the location of googleapis proto files")
+# $(error "env variable PROTOINC must be defined to the location of googleapis proto files")
```
Comment (renaissanceGeek) [inline grpc/Makefile:8]: Do you mean to leave this commented out?

**Review by renaissanceGeek — APPROVED** [review]
LGTM.  I'll admit, I don't know when someone should use java-lite vs the java build, but I assume a regular Java developer would know and find the distinction helpful.  (I am outdated--I haven't done Java development in many years and so never experienced Java-lite, only Java-very-heavy! 😉  )

File: grpc/Makefile (hunk 1 above)
Comment (tumarsal) [reply grpc/Makefile:8]: No sorry.

**Comment by happyalu** [comment]
I had some time to look at this today. The latest java sdk supersedes this work, closing this.

**Pull request closed without merging**